qiskit/result/models.py

```diff
@@ -22,35 +22,17 @@
 class ExperimentResultData:
     """Class representing experiment result data"""
 
-    def __init__(self, counts=None, snapshots=None, memory=None,
-                 statevector=None, unitary=None):
+    def __init__(self, **kwargs):
```

Comment:
Can we leave the old fields here and just add kwargs to the end? This will make it easier for existing users @jyu00

Reply:
Done @mtreinish